tests/data/test1915

```diff
@@ -30,7 +30,7 @@ lib%TESTNUMBER
 </tool>
 
 <command>
-http://%HOSTIP:%NOLISTENPORT/not-there/%TESTNUMBER
+http://%HOSTIP:%NOLISTENPORT/not-there/%TESTNUMBER --connect-timeout 1
```

Comment:
Since this is a libtest, the timeout handling needs to be done in the code. This is just the URL passed to the `<tool>`.

Reply:
Oh, of course. Silly me.